src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -39,12 +37,11 @@
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_PHONE + "91234567 ";
+    // for edit phone's issues, change p/phone to e/emergency_contact or keep it
```

Comment:
I think we should keep both. They will both be Phone objects, but with different prefixes.

Reply:
Yes, thank you!

Reply:
Yup, should keep both. "e/" for emergency contact, "p/" for employee's contact number.